lms/djangoapps/completion/handlers.py

```diff
@@ -21,9 +22,14 @@ def scorable_block_completion(sender, **kwargs):  # pylint: disable=unused-argum
     """
     if not waffle.waffle().is_enabled(waffle.ENABLE_COMPLETION_TRACKING):
         return
-    user = User.objects.get(id=kwargs['user_id'])
     course_key = CourseKey.from_string(kwargs['course_id'])
     block_key = UsageKey.from_string(kwargs['usage_id'])
+    block_cls = XBlock.load_class(block_key.block_type)
+    if getattr(block_cls, 'completion_method', 'scorable') != 'scorable':
```

Comment:
At https://github.com/edx/XBlock/blob/b0a6120208220c7ffffac5db189db03f7f26ee12/xblock/completable.py#L6-L12 , it seems like `scorable` is not a valid value, and what you mean by `scorable` is defined by `completion_method == XBlockCompletionMode.COMPLETABLE and has_custom_completion == True`. Or is it something different?

In any case, we should be consistent, and I'd prefer to bump the `XBlock` version as part of this PR so that we can use those constants instead of strings, to avoid inconsistencies like this.

Reply:
Yep.  I've been working in a haze this week, due to potty training one child while sleep training the other.  Thanks for catching this.

Reply:
As a side note, I see that the class is called XBlockCompletionMode (instead of the term Method, which we had been using.  I actually think I prefer that term.  I've been finding it difficult to write unambiguous documentation about completion methods that check the completion_method.

Reply:
@jcdyer Yeah, mode makes sense, but we currently have statements like this which use both terms:

```
completion_method = XBlockCompletionMode.COMPLETABLE
```

which is arguably worse than using one term or the other. Think it's not too late to change that to `completion_mode = XBlockCompletionMode.COMPLETABLE`?